src/components/AcceptedValuesField/AcceptedValuesField.js

```diff
@@ -53,18 +54,36 @@ export const AcceptedValuesField = ({
     name: optionTemplate ? updateValueWithTemplate(option, optionTemplate) : option.name,
   }));
 
+  const extendDataWithStatisticalCodeType = (arrToExtend, arrWithExtendedField) => {
+    const extendedData = sortCollection(arrToExtend, ['code']).map(item => {
+      const correspondRecord = arrWithExtendedField.find(itemAlt => itemAlt.id === item.statisticalCodeTypeId);
+
+      return {
+        ...item,
+        statisticalCodeTypeName: correspondRecord.name,
+      };
+    });
+
+    return sortCollection(extendedData, ['statisticalCodeTypeName']);
+  };
+
+  const mappedFns = { statisticalCodeTypeName: extendDataWithStatisticalCodeType };
+
   useEffect(() => {
-    if (wrapperSourceLink && wrapperSourcePath && isEmpty(acceptedValuesList)) {
-      fetchAcceptedValuesList(okapi, wrapperSourceLink, wrapperSourcePath)
-        .then(data => {
-          const acceptedValues = getAcceptedValuesObj(data);
-          const updatedListOptions = updateListOptions(data);
-
-          setListOptions(updatedListOptions);
-          setAcceptedValues(acceptedValues);
-        });
+    if (wrapperSources && isEmpty(acceptedValuesList)) {
+      const promises = wrapperSources.map(source => fetchAcceptedValuesList(okapi, source.wrapperSourceLink, source.wrapperSourcePath));
```

Comment:
Spread it over two lines, please. Also, you can use the destructuring assignment here, but it is at your discretion.
```suggestion
      const promises = wrapperSources
                .map(source => fetchAcceptedValuesList(okapi, source.wrapperSourceLink, source.wrapperSourcePath));
```

Reply:
done